_posts/2023-02-17-7th-workshop-announcement.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: OGGM - Global Glacier Modeling Workshop 2023
+subtitle: 28 August - 1 September, Edinburgh, Scotland
+author: Beatriz Recinos, Anouk Vlug, Dan Goldberg, Fabien Maussion and Ben Marzeion
+date: 2023-02-17T00:00:00
+tags: workshop
+---
+
+### What is it ?
+
+**The workshop is the 7<sup>th</sup> in a series that has brought together the community of users 
+and developers of the Open Global Glacier Model ([OGGM](oggm.org)). This year's workshop 
+is organized by colleagues based at The University of Edinburgh. It is open to anybody interested in global glacier modeling with focus on active developers and users of OGGM. The primary goal is to discuss the challenges 
+and solutions in modeling glaciers at large scales with OGGM, to learn from each other, and to develop 
+ideas and visions for the advancement of the model. The workshop will include scientific 
+presentations, hands-on coding sessions, discussion/brainstorming sessions, and generally 
+be held in an informal manner (with a possible hike to a Scotish National park).**
+
+### Agenda
+
+The workshop will include:
+
+1. <b>Scientific presentations</b> by the participants (ca. 15-20 min) on topics related to the theme of the workshop. 
+These can include the state, recent developments and results of glacier modeling with OGGM.
+2. <b>Discussion/brainstorming sessions</b> focused on topics of interest, such as future developments and visions for the model.
+3. <b>Hands-on sessions</b> in smaller groups including:
+    - Beginner tutorials: these sessions are for current and future OGGM users to learn how to set-up and customize a model run.
+    - Advanced tutorials: on topics such as <i>How to contribute to OGGM</i>, <i>use OGGM with your own data</i>, or <i>use parts of OGGM with another model</i>.
+    - Q&A session: on model problems and possible solutions.
```

Comment:
This time we would like to have the beginner hands-on sessions on the first 1.5 days on the workshop, so that people can get up to speed and better understand what is said in the later sessions. Probably worth mentioning here